src/components/menus/branch-menu.js

```diff
@@ -181,7 +179,7 @@ const withBranchMenu =
 
         return (
             <Menu
-                className={classes.menu}
+                sx={styles.menu}
```

Comment:
it does not seem to exists -> to remove ?

Reply:
Thanks. Fixed.